tests/test_smab.py

```diff
@@ -802,3 +829,82 @@ def test_smab_mo_cc_from_state(state):
     # Ensure get_state and from_state compatibility
     new_smab = globals()[smab.get_state()[0]].from_state(state=smab.get_state()[1])
     assert new_smab == smab
+
+
+########################################################################################################################
+
+
+# Smab with epsilon-greedy super strategy
+
+
+@given(
+    st.integers(min_value=1),
+    st.integers(min_value=1),
+)
+def test_can_instantiate_epsilon_greddy_smab_with_params(a, b):
+    s = SmabBernoulli(
+        actions={
+            "action1": Beta(n_successes=a, n_failures=b),
+            "action2": Beta(n_successes=a, n_failures=b),
+        },
+        epsilon=0.1,
+        default_action="a1",
+    )
+    assert (s.actions["action1"].n_successes == a) and (s.actions["action1"].n_failures == b)
+    assert s.actions["action1"] == s.actions["action2"]
+
+
+def test_epsilon_greddy_smab_predict():
+    s = SmabBernoulli(
+        actions={
+            "a0": Beta(),
+            "a1": Beta(n_successes=5, n_failures=5),
+            "forb_1": Beta(n_successes=10, n_failures=1),
+            "best": Beta(n_successes=10, n_failures=5),
+            "forb_2": Beta(n_successes=100, n_failures=4),
+            "a5": Beta(),
+        },
+        epsilon=0.1,
+        default_action="a1",
+    )
+    forbidden_actions = set(["forb_1", "forb_2"])
+
+    _, _ = s.predict(n_samples=1000, forbidden_actions=forbidden_actions)
+
+
```

Comment:
If you can for all tests below try to avoid hardcoded values in the test body. But define them in the signature of the function (either with hypotesis or not)

Reply:
Please be more specific. If you refer to n_samples = 1000, I did it similarly to test_smab_mo_predict and other tests. Only difference is that I did not define a variable n_samples=1000 and passed it to the function

Reply:
Just a minor comment: in `test_epsilon_greedy_smab_predict()` you do not define n_samples=1000 as variable, while in `test_epsilon_greddy_smab_mo_predict()` you define it in the first line of the function.
You can use the same approach (whatever you want) to keep all the code with the same style.